source/TradingPanel.cpp

```diff
@@ -209,10 +204,14 @@ void TradingPanel::Draw()
 	if(showProfit)
 		font.Draw("Profit", Point(MIN_X + PROFIT_X, FIRST_Y), selected);
 
+	canSellOutfits = outfitCargo &&
+		(player.GetPlanet()->HasOutfitter() || Preferences::Has("'Sell Outfits' without outfitter"));
 	Information info;
-	if(sellOutfits)
+	if(canSellOutfits)
 		info.SetCondition("can sell outfits");
-	else if(player.Cargo().HasOutfits() || canSell)
+	if(minableCargo)
```

Comment:
For some reason, I can do this:
![zero](https://github.com/user-attachments/assets/6162cb5c-5247-4be5-9513-3bde12567c4e)
And I don't think it's a rounding error, because it can happen immediately after reloading the save.

Reply:
Can you give me a save file?

Reply:
Happens any time you don't have anything to sell and you click Sell Specials.